src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Nexylan packages.
+ *
+ * (c) Nexylan SAS <contact@nexylan.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nexy\Gandi\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Jérôme Pogeant <p-jerome@hotmail.fr>
+ */
+final class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('nexy_gandi');
+
+        $rootNode
+            ->children()
+            ->scalarNode('server_url')
+            ->end()
+            ->scalarNode('api_key')
+            ->end()
```

Comment:
I thing a `end()` is missing.

Is it tested? Maybe you can use this library and write some unit test: https://packagist.org/packages/matthiasnoback/symfony-config-test

I can setup Travis for you.